makefile

```
@@ -17,7 +17,7 @@ create_image:
 .phony: create_image
 
 run_image:
-	docker run -p 80:80 --rm --env-file=./.env ${MODULE}:${VERSION_NAME}
+	docker run --device=/dev/ttyUSB0  --rm --env-file=./config.env ${MODULE}:latest
```

```suggestion
	docker run --device=/dev/ttyUSB0  --rm --env-file=./.env ${MODULE}:${VERSION_NAME}
```
we use `v1.0.0` as tag now

---

you added an extra space at the end of the line, this is actually prevented by using the settings from `.editorconfig`. if you are using VS code you can install this [extension](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig) for example

---

o i already installed it